src/main/java/mate/academy/springbookapp/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,21 @@
 package mate.academy.springbookapp.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank(message = "Title may not be blank")
     private String title;
+    @NotBlank(message = "Author may not be blank")
     private String author;
+    @NotBlank(message = "ISBN may not be blank")
     private String isbn;
```

Comment:
If json from FE contains 'Title' field it would be confused to see massage "Title may not be blank")
```suggestion
    @NotBlank(message = "title may not be blank")
    private String title;
    @NotBlank(message = "author may not be blank")
    private String author;
    @NotBlank(message = "isbn may not be blank")
    private String isbn;
```